modules/local/chrom_sizes.nf

```diff
@@ -0,0 +1,28 @@
+process CHROM_SIZES {
+    label 'process_nompi'
```

Comment:
here you have 2 options for tag:
if you pass meta, the meta.id
if meta doesn't make sense, the genome simple name